plugin/api.go

```diff
@@ -254,6 +254,9 @@ type API interface {
 	// Minimum server version: 5.3
 	ReadFile(path string) ([]byte, *model.AppError)
 
+	// GetEmojiImage returns the emoji image.
```

Comment:
Can you document the minimum version required here?

```suggestion
	// GetEmojiImage returns the emoji image.
	//
	// Minimum server version: 5.6
```